src/main/java/org/sonar/plugins/stash/client/StashClient.java

```diff
@@ -35,17 +35,20 @@
   private final int stashTimeout;
   private AsyncHttpClient httpClient;
 
-  private static final String REST_API = "/rest/api/1.0/";
-  
-  private static final String USER_API = "{0}users/{1}";
-  private static final String REPO_API = "{0}projects/{1}/repos/{2}/";
-  private static final String PULL_REQUESTS_API = REPO_API + "pull-requests/";
-  private static final String PULL_REQUEST_API = PULL_REQUESTS_API + "{3}";
-  private static final String COMMENTS_PULL_REQUEST_API = PULL_REQUEST_API + "/comments";
-  private static final String COMMENT_PULL_REQUEST_API = COMMENTS_PULL_REQUEST_API + "/{4}?version={5}";

```

Comment:
Could we remove all the query params from the string constants and use [`addQueryParams()`](http://static.javadoc.io/org.asynchttpclient/async-http-client/2.1.0-alpha1/org/asynchttpclient/RequestBuilderBase.html#addQueryParam-java.lang.String-java.lang.String-)?